### ISSUE TITLE
Practical guidance for adopting open source tooling and open coding

### ISSUE THREAD

**Issue by wbryant** [nhs-r-community/nhs-r_vision#1]
Part of [what-is-the-need-in-the-system](https://github.com/nhs-r-community/nhs-r_vision#what-is-the-need-in-the-system-given-the-change-in-the-health-and-care-landscape) that could be of great benefit would be a knowledgebase/playbook for adopting and supporting open source tools and mindsets in the NHS.  For instance someone shared some great resources addressing (I think) getting authorisation for R installation and running open source tools.

It's quite a broad topic so could be honed for NHS-R but I think it's a great community to be involved in that sort of work.

**Comment by wbryant** [comment]
The content of this is addressed somewhat in the following section, but not how such information would be delivered.

**Comment by ChrisBeeley** [comment]
Kind of thinking aloud here, a collection of relevant links for reference

* https://nhsx.github.io/AnalyticsUnit/experiences.html
* https://www.nhsx.nhs.uk/blogs/data-saves-lives-building-and-skilling-nhs-analytics-community/
* https://nhsx.github.io/AnalyticsUnit/toolstackrequests.html
* https://nhsx.github.io/AnalyticsUnit/codeintheopen.html

General open code stuff

* https://github.com/nhsx/open-source-policy
* https://www.gov.uk/government/publications/data-saves-lives-reshaping-health-and-social-care-with-data-draft/data-saves-lives-reshaping-health-and-social-care-with-data-draft

**Comment by ChrisBeeley** [comment]
If you would like to contribute to this issue please make a PR here https://github.com/nhs-r-community/nhs-r_vision/blob/main/content/docs/open_guidelines/_index.md

**Comment by wbryant** [comment]
I'm completely out of sync, I'm working on creating content on https://github.com/nhs-r-community/statements-on-tools - is there still a need to update the vision statement?

**Comment by ChrisBeeley** [comment]
You are right, there is a bit of overlap between these two repos. It's the nature of anarchic collaborative collaboration I guess.

I suspect statement on tools can feed straight in to this, so just carry on and we'll bring it all together in due course.

Thank you 🙏
